test/e2e/hubble/scenario.go

```diff
@@ -0,0 +1,56 @@
+package hubble
+
+import (
+	"net/http"
+
+	k8s "github.com/microsoft/retina/test/e2e/framework/kubernetes"
+	"github.com/microsoft/retina/test/e2e/framework/types"
+)
+
+func ValidateHubbleRelayService() *types.Scenario {
```

Comment:
need to validate that relay is actually running, not just exisiting, similar to how we have to port forward to 8080 and check for http response for Hubble UI, per https://github.com/microsoft/retina/issues/921

Reply:
pod running state is validated in waitforpodready(), as for the forward step, will skip for now until we decide to add hubble cli install step in e2e since hubble relay pod forward is for HTTP/0.9 and couldn't get response from the web